setup.py

```diff
@@ -1,11 +1,22 @@
+from pathlib import Path
+from pip.req import parse_requirements
 from setuptools import find_packages, setup
 
+
+install_requires = []
+for path in Path('./').rglob('requirements.txt'):
+    _reqs = parse_requirements(str(path.absolute()), session='hack')
+    reqs = [str(r.req) for r in _reqs]
+    install_requires.extend(reqs)
+
+
 config = {
     "description": "Single function scripts to move and manage your data across multiple vendors.",
     "author": "Shipyard Team",
     "url": "https://github.com/shipyardapp/starter-blueprints",
     "author_email": "tech@shipyardapp.com",
     "packages": find_packages(),
+    "install_requires": install_requires,
```

Comment:
I'm good with adding this. 

I think this may not be needed because of the way that Blake has the Blueprint running inside of Shipyard. But it would be important for local install and testing. 

Blake and I talked and we are going to leave this in. But, it may change in the future depending on how we configure Shipyard and get all of these Blueprints to work nicely.